.travis.yml

```diff
@@ -0,0 +1,21 @@
+os:
+  - linux
+dist: bionic
+language: python
+
+stages:
+- name: test
+
+jobs:
+  include:
+    - &test
+      stage: test
+      name: Run linting and unit tests
+      python: 3.6
+      script:
+        - flake8
+        - coverage run --source broker -m pytest
+    - <<: *test
+      python: 3.7
+    - <<: *test
+      python: 3.8
```

Comment:
Just out of curiosity: does this build for Python 3.9 as well? At least the latest version of FOCA does...